staging/src/kubevirt.io/application-aware-quota-api/libsidecar/sidecar.go

```diff
@@ -20,7 +20,7 @@ const (
 )
 
 type SidecarCalculator interface {
-	PodUsageFunc(podToEvaluate *corev1.Pod, existingPods []*corev1.Pod) (corev1.ResourceList, error, bool)
+	PodUsageFunc(podToEvaluate *corev1.Pod, existingPods []*corev1.Pod) (corev1.ResourceList, bool, error)
```

Comment:
Maybe we can drop the bool return param?

```suggestion
	PodUsageFunc(podToEvaluate *corev1.Pod, existingPods []*corev1.Pod) (*corev1.ResourceList, error)
```

This way the user can just return nil in this case.

Another thought is to add named return params to explain the bool's purpose:
```suggestion
	PodUsageFunc(podToEvaluate *corev1.Pod, existingPods []*corev1.Pod) (updatedResources corev1.ResourceList, isPodAffected bool, err error)
```

But that's just a thought :) we can think about it in a follow-up

Reply:
Intuitively, I think it will be easier for users to understand the application's belonging to an app if we use a boolean. However, if we receive feedback that nil is better, I'm open to changing this. Thanks!